src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -164,7 +164,7 @@ private static string GetMemberElementName(FunctionDescriptor member)
             {
                 case FunctionType.Constructor:
                     // XML documentation uses slightly different constructor names
-                    memberName = memberName.Replace(".ctor", "#ctor");
+                    memberName = member.ClassName + ".#ctor";
```

Comment:
`string memberName = member.ClassName + "." + member.FunctionName;`
That is how `memberName` creates.
Name for constructors is a bit different.
E.g. Namespace.Class.#ctor